src/utils/find.ts

```diff
@@ -17,6 +17,8 @@ function matches(node: VNode, selector: FindAllComponentsSelector): boolean {
   }
 
   if (typeof selector === 'object' && typeof node.type === 'object') {
+    if (selector === node.type) return true
```

Comment:
what does this do? in what cases does this eval to true?

Reply:
If you pass the component definition and it's identical to the one the component was created with. Like imported SFC

Reply:
Right, ok. so many ways to make a component in vue haha